asmrun/arm.S

```diff
@@ -17,7 +17,7 @@
 
         .syntax unified
         .text
-#if defined(SYS_linux_eabihf) && defined(MODEL_armv6)
+#if (defined(SYS_linux_eabihf) || defined(SYS_netbsd)) && defined(MODEL_armv6)
```

Comment:
could we define something more generic for the `SYS_linux_eabihf || SYS_netbsd`?  This is going to get more elaborate as we add other hardfloat BSDs, so not scattering it through `arm.S` would be nice...